examples/polyfill.html

```diff
@@ -13,19 +13,19 @@
     }
```

Comment:
Since we dropped default support for IE11 in https://github.com/github/image-crop-element/pull/12 this file is not relevant anymore (reference to this file was removed in that PR) and can be removed.

The build file wouldn't work on IE11 without the class transformation which users will need to compile for themselves.